units/suspend-then-hibernate.target

```diff
@@ -0,0 +1,16 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Suspend; Idle into hibernate
```

Comment:
That was already before this PR, I just noticed it: this Description does not make sense — "Idle into hibernate"?

Something wrong happens in the rename:
before:
units/suspend-to-hibernate.target
units/systemd-suspend-to-hibernate.service.in
after:
units/suspend-then-hibernate.service
units/suspend-then-hibernate.target
units/systemd-suspend-then-hibernate.service.in

Reply:
(Please make the change of the Description a separate commit.)

Reply:
Thanks for catching that, I thought something seemed off and was confused why git wasn't renaming one of the files.

I've fixed both of these.